Cargo.toml

```diff
@@ -22,7 +22,7 @@ maintenance = { status = "actively-developed", branch = "main" }
 
 [dependencies]
 # Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
-libc = { version = "0.2", default-features = false }
+libc = { version = "0.2.160", default-features = false }
```

Comment:
These lines are at odds with each other, as mozilla-central currently depends on `libc` `v0.2.158`. We will need to bump mozilla-central's `libc` version before we can merge here.

https://searchfox.org/mozilla-central/rev/d4235354139ae0da3b15b89943b140b510a74f2f/Cargo.lock#3533-3537